### DISCUSSION TITLE
GitHub Projects Cleanup?

### DISCUSSION THREAD

**Discussion by kingthorin** [community/community#23641] [New to GitHub]
<p>If I have a repo where we have Projects enabled, but then we decide to disable them (there’s some debate among the team whether we should use them or not). Does whatever it currently setup continue to exist? For example if I turn it off now with two projects half done (or whatever) when (if) we turn it back on in 6mo will those two projects and relevant state details still be there?</p>

**Comment by canuckjacq** [comment]
<p>If you uncheck the projects option in your repository settings or in your org settings, it seems to effectively just hide your projects/references to projects. When you check it again, that comes back and your project is still there, and issues linked to projects retain that too.</p>

**Comment by kingthorin** [comment]
<p>Thanks, I did try that in a junk repo with the enabling/disable just seconds apart. But, wanted to ensure that there wasn’t some midnight or monthly job on the backend (GitHub) that might “clean” it up.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>I have found you official confirmation:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">
      <a href="https://github.blog/2017-03-24-disabling-projects/" target="_blank" rel="noopener" title="03:24AM - 25 March 2017">The GitHub Blog – 25 Mar 17</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://github.blog/wp-content/uploads/2017/03/3c947f8a-1081-11e7-8fab-6106428215f2.png?fit=2400%2C1260" class="thumbnail" width="690" height="362"></div>

<h3><a href="https://github.blog/2017-03-24-disabling-projects/" target="_blank" rel="noopener">Disabling projects - The GitHub Blog</a></h3>

<p>Not every team manages their work on GitHub in the same way. Now you can disable repository and organization-wide Projects if you’re not using them. Users with admin privileges on a repository can disable Projects</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<blockquote>
<p>Projects can be re-enabled at any time, at which point all previously-disabled projects will be restored exactly as you left them.</p>
</blockquote>
<p>:tada:</p>
